### DISCUSSION TITLE
Daily 13/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#56] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer participé de la clase de testing y estuve leyendo los apuntes de app móviles de la plataforma.
Hoy voy a estar trabajando sobre las correcciones que vimos ayer para hacer en el diagrama de clases. También estaré empezando con  el segundo ejercicio de app móviles y preparando la tarea para la clase de inglés que tengo a la noche.

**Comment by MarielaGomez** [comment]
Ayer tuve clase de testing donde abordamos mejor el tema de la entrega del trabajo práctico , adelanté lectura del marco teórico, resolví cuestionarios de ciberseguridad, ahora continúo desarrollando la UML de los casos de uso

**Comment by Valentino399** [comment]
Ayer estaba diseñando la home de la aplicación con la herramienta Figma

**Comment by BraianTroncoso** [comment]
Estuve leyendo los apuntes, completé 2 de los cuestionarios de app moviles. También entregué el pdf sobre testing. Me puse a estudiar un poco de android y consultar unas dudas sobre el proyecto con otros colegas.
Tambien realicé junto con nati y marce la script de la db incluyendo el diagrama.
